webrender_api/src/display_list.rs

```diff
@@ -699,6 +699,22 @@ impl DisplayListBuilder {
         )
     }
 
+    fn push_item_with_scrollinfo(
```

Comment:
This should probably be called either `push_item_with_scroll_info` or `push_item_with_clip_scroll_info.`